Atomic/containers.py

```diff
@@ -187,9 +187,9 @@ def _check_filters():
                 return True
             for f in self.args.filters:
                 _filter, _ = f.split('=', 1)
-                if _filter not in [x for x in self.FILTER_KEYWORDS]:
+                if _filter not in self.FILTER_KEYWORDS.keys():
                     raise ValueError("The filter {} is not valid.  "
-                                     "Please choose from {}".format(_filter, [x for x in self.FILTER_KEYWORDS]))
+                                     "Please choose from {}".format(_filter, self.FILTER_KEYWORDS.keys()))
```

Comment:
hm.. I get exactly the same output, also:

```
>>> h = {'a' : 1, 'b' : 2}
>>> print("message {}".format(h.keys()))
message ['a', 'b']
>>> print("message {}".format([x for x in h]))
message ['a', 'b']
```

What version of Python are you using?

```